### DISCUSSION TITLE
a/c minutes doesnt work

### DISCUSSION THREAD

**Discussion by user7590** [TobiasLaross/Hass-Lynk-Co#37] [Q&A] [answered]
first of all: great work!

when i start the climate control, it works some 2/3 minutes, then it stops. the number of minutes do not have any effect. any thoughts?
The card looks like this:

type: tile
entity: binary_sensor.lynk_co_pre_climate_active
tap_action:
  action: call-service
  service: lynkco.start_climate
  target: {}
  data:
    climate_level: HIGH
    duration_in_minutes: 15
icon_tap_action:
  action: call-service
  service: lynkco.stop_climate
  target: {}
vertical: false
show_entity_picture: true
name: 'Climate Control is:'

**Comment by TobiasLaross** [comment]
Was the car warm when you started?

When the car is plugged in only the start_climate will work is should be renamed to pre_heating, it will heat the car up to pre-set temperature, so if the car is already warm it will turn itself off.

if the car is not plugged in and experimental features flag is on you can start_engine which will cool or hear the car to the temperature set in the car. But this is not supported by Lynk.

**Comment by user7590** [comment]
Thanks Tobias!
In my case i'm away from the car, the car is not plugged in, and i want to cool the car for f.e. 15 minutes. So what is your advice how to handle this in the dashboard?

**Comment by TobiasLaross** [comment, marked as answer]
If you want to use experimental flag you can use start_engine for this use case

**Comment by user7590** [comment]
Still, to get back to this Tobias: it (ac only) -does- work for about 2 minutes (so without engine on and without charging).......isnt there any way to change this period?

**Reply by TobiasLaross** [reply]
I think that is a backend/fw bug that starts the AC, I thought it didn’t work for more than 15s though. But no it is not possible to change that period.

**Reply by user7590** [reply]
ah, that is what i thought yes. And it ight even be 15s yes, it's just that the next refresh shows that its off again, which is a few mins most of the time here. thankd again
